sources/logdog/static/aws-ecs-1

```diff
@@ -0,0 +1,5 @@
+http get http://localhost:51678/v1/tasks
+sh docker-daemon.json cat /etc/docker/daemon.json
+sh docker-info docker info
+sh ecs-agent-state.json cat /var/lib/ecs/data/ecs_agent_data.json
+sh ecs-config.json cat /etc/ecs/ecs.config.json
```

Comment:
Maybe a `file` mode to handle the common case of copying files like this?

Reply:
[aa7d592](https://github.com/bottlerocket-os/bottlerocket/compare/60e5d3c8e65c58170e8f72d944a80be11ea098e0..aa7d59202dad5161a9dc8aff3995f1fbe0d4c706)